test/elixir/test/partition_view_update_test.exs

```diff
@@ -0,0 +1,73 @@
+defmodule PartitionSizeTest do
```

Comment:
I believe this should be `PartitionViewUpdateTest` so it doesn't conflict with another `PartitionSizeTest` in partition_size_test.exs:
```
== Compilation error in file ../src/couchdb/test/elixir/test/partition_view_update_test.exs ==
** (CompileError) ../src/couchdb/test/elixir/test/partition_view_update_test.exs:1: cannot define module PartitionSizeTest because it is currently being defined in /Users/jay/proj/ibm/db/src/couchdb/test/elixir/test/partition_size_test.exs:1
    (stdlib) erl_eval.erl:670: :erl_eval.do_apply/6
    (elixir) lib/code.ex:677: Code.require_file/2
    (elixir) lib/kernel/parallel_compiler.ex:201: anonymous fn/4 in Kernel.ParallelCompiler.spawn_workers/6
make: *** [elixir-test] Error 1
returned 2
```

Reply:
Derp. Good catch. That replication test takes too long I was only ever running them in isolation.